### PR TITLE
LPD-42596 Display all section in dashboard page of CMS accelerator

### PR DESCRIPTION
**Pull request by jkappler**
Display all section in dashboard page of CMS accelerator

# What is this trying to solve?

[Link to ticket](https://issues.liferay.com/browse/LPD-42596)

This is the first version of the section that will display all entries in the new CMS acceletaror. For it to work, two feature flags are needed:

LPD-17809, which enables creating the CMS site
LPD-11232, which enables the search headless API

When the portal starts, navigate to the CMS site: /web/cms and the items that belong to all sites will be shown in a frontend data set

### CODE REVIEW

File: modules/apps/frontend-data-set/frontend-data-set-taglib/bnd.bnd [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Bundle-Name: Liferay Frontend Data Set Taglib
 Bundle-SymbolicName: com.liferay.frontend.data.set.taglib
 Bundle-Version: 1.0.48
+Export-Package:\
+	com.liferay.frontend.data.set.taglib.servlet.taglib
```
Comment (jkappler) [inline modules/apps/frontend-data-set/frontend-data-set-taglib/bnd.bnd:5]: @dsanz, exporting the package as we mentioned

**Comment by jkappler** [comment]
ci:test:sf

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:forward

**Comment by adolfopa** [comment]
@jkappler The semver error seems to be related to the changes in the bnd.

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:forward:force

**Pull request closed without merging**